virus/Thogotovirus_aidenensis.md

```diff
@@ -0,0 +1,148 @@
+# {Q1A Paper Title}
```

Comment:
All the sub-sections in `{ }` are variables, please change the titles and sub-section names to fit a "Virus Report", not as a checklist of the assignment.